security/keycloak-multitenancy/README.md

```diff
@@ -4,9 +4,9 @@ This example shows how the application and health endpoints requiring different
 
 ## Start Keycloak
 
-### Keycloak Swarm Server
+### Keycloak Thorntail Server
 
-Download the latest Swarm Keycloak standalone server jar, for example, 2018.5.0 version:
+Download the latest Thorntail Keycloak standalone server jar, for example, 2018.5.0 version:
```

Comment:
The version number should probably be updated, too, as there's no Thorntail 2018.5.0.